set_goenv.sh

```diff
@@ -83,7 +83,7 @@ set_goenv() {
 }
 
 print_ldflags() {
-    VersionStr="$(git describe)"
+    VersionStr="$(go run release/release.go get-version)"
```

Comment:
make sure `--version` is getting its version info from the same place as our packaging code